cmd/aperture-agent/config/types.go

```diff
@@ -63,8 +63,7 @@ type BatchPostrollupConfig struct {
 	SendBatchMaxSize uint32 `json:"send_batch_max_size" validate:"gte=0" default:"100"`
 }
 
-// CustomMetricsConfig defines receivers, processors and single metrics pipeline,
-// which will be exported to the controller prometheus.
+// CustomMetricsConfig defines receivers, processors, and single metrics pipeline which will be exported to the controller prometheus.
```

Comment:
@openai can you update the documentation for `CustomMetricsConfig` to show how the user can provide OpenTelemetry collector configuration?

Example configuration that a user can provide -
```
otel:
      custom_metrics:
        rabbitmq:
          pipeline:
            processors:
            - batch
            receivers:
            - rabbitmq
          processors:
            batch:
              send_batch_size: 10
              timeout: 10s
          receivers:
            rabbitmq:
              collection_interval: 10s
              endpoint: http://my-release-rabbitmq-0.my-release-rabbitmq-headless.default.svc.cluster.local:15672/
              password: secretpassword
              username: admin
```

Reply:
Can you add more details to these comments using your knowledge of the OpenTelemetry collector configuration?